docs/en/upgrade.md

```diff
@@ -153,6 +153,15 @@ The solution automatically and asynchronously upgrades the views and materialize
     DROP PROCEDURE "<app-id>".sp_clear_item_and_user();
     ```
 
+## Post-Upgrade Actions II (only applicable for upgrading to V1.2.0 from earlier version)
```

Comment:
V1.2.0 -> v1.2..0

keep consistency in the doc

Reply:
Refactored